vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml

```diff
@@ -26,6 +30,11 @@
     dest: "{{ root_folder }}/install/pentaho/foia-reports-dw-{{ foia_analytical_reports_version }}"
     creates: "{{ root_folder }}/install/pentaho/foia-reports-dw-{{ foia_analytical_reports_version }}/foia-dw1.kjb"
 
+##########################################################
+# Configure the Kettle job files and run the Kettle job.
+# This will create and populate the OLAP tables.
+##########################################################
```

Comment:
@david-oc-miller , minor one, it will populate Data Warehouse tables, not OLAP.

Reply:
@marjan4jc are the data warehouse tables not OLAP cubes?

Reply:
@david-oc-miller , yes they are dw tables, the OLAP cube is defined in a mondrian schema file.